doc/ref_model/chapters/chapter04.md

```diff
@@ -280,45 +280,77 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
+
+* Basic: Minimum number attributes for Virtual Server Hardware configuration
+
+* High Performance: Detailed Hardware configuration of a virtual server. It allows a deep level configuration (e.g. Specific configuration parameters for Hardware acceleration)
 
->_*Note:*_ Customised (Parameterized) Flavours can be used in concession by operators and, if needed, are created using TOSCA, HEAT templates, and/or VIM APIs.
 
 <a name="Table4-13"></a>
 
-| .conf                  | vCPU ("c") <sup>2)</sup> | RAM ("r") <sup>2)</sup> | Local Disk ("d") <sup>3)</sup>| Bandwidth |
-|------------------------|--------------------------|-------------------------|------------------|----------------------|
-| .tiny                  | 1                        | 512 MB                  | 1 GB             | 1 Gbps               |
-| .small                 | 1                        | 2 GB                    | 20 GB            | 1 Gbps               |
-| .medium                | 2                        | 4 GB                    | 40 GB            | 1 Gbps               |
-| .large                 | 4                        | 8 GB                    | 80 GB            | 1 Gbps               |
-| .2xlarge <sup>1)</sup> | 8                        | 16 GB                   | 160 GB           | 1 Gbps               |
-| .4xlarge <sup>1)</sup> | 16                       | 32 GB                   | 320 GB           | 1 Gbps               |
-| .8xlarge <sup>1)</sup> | 32                       | 64 GB                   | 640 GB           | 1 Gbps               |
-
-<p align="center"><b>Table 4-13:</b> Predefined Compute Flavours.</p>
-
-**1)** These compute Flavours are intended to be used for transitional purposes and workload vendors are expected to consume smaller Flavours and adopt microservices-based designs for their workloads.<br>
-**2)** In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.<br>
-**3)** The "local disk" is an ephemeral disk that provides storage for the life of a VM/Pod, and can either be provided by storage devices housed within the physical server on which the VM/Pod is running, or from an external storage device such as SAN or NFS.
+
+| Ref | Attribute | Unit | (B)asic/(H)igh Performance | Presence |
+|----------|--------------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------|
+| i.fl.001 | Flavor ID             | integer       | B/H           | Required |
```

Comment:
Is the fl in the reference for "flavour"? If yes, then many items in this table are not in flavours but in profiles. Please note that for workloads OpenStack uses "Flavours" to also specify the profile but in RM that is not the case.

Reply:
@pgoyal01 This is a very good point that we need to clarify. We're in favour of using workloads in RM, agnostic to the Infrastructure, and leave to the RA1 (Flavor) and RA2 (workload) the data model definition. Therefore, we should define it in a granular way (in RM). Is that OK?